### ISSUE TITLE
File Limit Request: Coupled VAE - 400 MB

### ISSUE THREAD

**Issue by jkclem** [pypi/support#2505]
### Project URL

https://test.pypi.org/project/Coupled-VAE/

### Does this project already exist?

- [X] Yes

### New Limit

400

### Update issue title

- [X] I have updated the title.

### Which indexes

PyPI, TestPyPI

### About the project

There is no data, only code, so I do not have any ideas on how to reduce size.

### Reasons for the request

I need to update the list of dependencies and cannot do that because of the size limit.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by cmaureir** [comment]
Hey @jkclem :wave: thanks for your report.
I'm a bit confused with the role of PyPi and your project. Because you mentioned you only have code, I got curious and looked at the repository, and I noticed that you have no mention of setuptools, distutils, or anything that would seem that you are creating a wheel for PyPi. I did see many Python Notebooks, which might increase a wheel size, but I'm not sure what's included.
Could it be that maybe you have changes or file in a different place than https://github.com/Photrek/Coupled-VAE ?
